sdk/nodejs/src/client.ts

```diff
@@ -50,6 +51,9 @@ export class DVCClient {
         }
 
         const initializePromise = importBucketingLib({ options, logger: this.logger })
+            .catch((bucketingErr) => {
+                throw new UserError(`Error importing bucketing library: ${bucketingErr}`)
```

Comment:
does this obscure the stack trace of the original error?

Reply:
added support for creating it from `Error` objects

Reply:
you'd have to pass the error in here then